ts/parts/SVGLabel.ts

```diff
@@ -0,0 +1,548 @@
+/* *
+ *
+ *  (c) 2010-2020 Torstein Honsi
```

Comment:
Not related to the PR itself, just a note: can we automate years? For example `annotations` have 2009-2017 (?) :) For example `@product.copyrightYears@`

Reply:
Yes we should. I don't think we need a variable for it though. We can have the build process look for actual touch dates in the files for example. I'm closing this comment and we can pick it up in a separate issue.